### PR TITLE
Update to v1.4.230

### PR DESCRIPTION
**Pull request by JBBgameich**

### CODE REVIEW

File: info.mumble.Mumble.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 id: info.mumble.Mumble
-rename-desktop-file: mumble.desktop
 rename-icon: mumble
+rename-desktop-file: org.mumble_voip.mumble.desktop
```
Comment (hfiguiere) [inline info.mumble.Mumble.yml:3]: you probably need to add `rename-appdata-file: org.mumble_voip.mumble.appdata.xml` here as well. Which would solve the build failure.

File: info.mumble.Mumble.yml [hunk 2]
```diff
@@ -70,37 +70,26 @@ modules:
         tag: v0.8
         commit: f060abee2807c943821d88839c013ce15db17b58
 
+  - name: poco
+    buildsystem: cmake
```
Comment (hfiguiere) [inline info.mumble.Mumble.yml:74]: `cmake-ninja` tends to be prefered, it's faster.

File: info.mumble.Mumble.yml (hunk 1 above)
Comment (JBBgameich) [reply info.mumble.Mumble.yml:3]: Thank you for the help

**Comment by hfiguiere** [comment]
the id in the appdata is wrong: `<id>org.mumble_voip.mumble</id>` and should be changed.

**Comment by JBBgameich** [comment]
Yes, I decided to contact upstream about the appstream id first: https://github.com/mumble-voip/mumble/issues/5441

**Comment by hfiguiere** [comment]
They should change it, but in the mean time, this is necessary

**Comment by JBBgameich** [comment]
I'm working on a quick workaround

**Comment by JBBgameich** [comment]
Okay, that didn't work. I guess I'll revert back to our custom appstream metadata when I get back home later today.

**Comment by hfiguiere** [comment]
the failure is:
```
builddir/files/share/appdata/info.mumble.Mumble.appdata.xml: FAILED:
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
• tag-missing           : <release> required
Validation of files failed
```

ie their appdata is incomplete.

**Comment by hfiguiere** [comment]
what we do here is patch it. And submit upstream.

**Comment by Krzmbrzl** [comment]
> what we do here is patch it. And submit upstream.

Yes please - I created the upstream file(s) but with close to zero knowledge of what I am actually doing. So if something (like the appstream ID) looks weird or out of place to you, assume that that's because I just don't have a clue about this stuff and patches fixing that are definitely welcome :D

**Pull request merged**